hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -159,6 +159,8 @@ objects:
           mediatype: ''
         publisher: Red Hat
         sourceType: grpc
+        grpcPodConfig:
```

Comment:
Does this work on clusters < 4.14? AFAIK, this will be rejected on the pre-validation.

Reply:
4.10 and 4.11 clusters should just ignore the setting.

Reply:
also, we can't wait for 4.11 to go EOL because operators can't deploy to 4.14 clusters without this setting.

Reply:
Learned that this is configurable, but unknown fields are ignored by default:

```
this is configurable and we haven’t been setting this for our operator CRDs and also applies to the CatalogSource CRD, so unknown fields work
❯ k explain crd.spec.versions.schema.openAPIV3Schema.x-kubernetes-preserve-unknown-fields
GROUP:      apiextensions.k8s.io
KIND:       CustomResourceDefinition
VERSION:    v1

FIELD: x-kubernetes-preserve-unknown-fields <boolean>

DESCRIPTION:
    x-kubernetes-preserve-unknown-fields stops the API server decoding step from
    pruning fields which are not specified in the validation schema. This
    affects fields recursively, but switches back to normal pruning behaviour if
    nested properties or additionalProperties are specified in the schema. This
    can either be true or undefined. False is forbidden.
```